routes/submitForm.js

```diff
@@ -1,16 +1,81 @@
-/*************************
-TODO PART 1
-TODO find a module that can submit mysql requests
-TODO write a query request in the function below
-TODO make it transaction based
-
-TODO Part2
-TODO put the config info for the mysql server in the config
-TODO send the front end a success message on success and an error message with error code on fail
-TODO remove these comments!
-*************************/
-
-var router = function(req, res) {
-};
-
-module.exports = router;
\ No newline at end of file
+var mysql = require("mysql");
+var config = require("konfig")();
+
+var express = require("express");
+
+var router = express.Router();
+
+/*
+Usage:
+    localhost:50001/submitForm/jobID/Description/employeeName
+*/
+
+router.route("/:jobID/:Description/:employeeName").get(function(req, res) {
+    var databaseName = "impDB";
+
+    var tableName = "Entry";
+    var tableFields = "(jobID int, Description TEXT, employeeName TINYTEXT)";
+
+    var id = req.params.jobID;
+    var desc = req.params.Description;
+    var name = req.params.employeeName;
+    var values = req.params.values || "(" + id + ", '" + desc + "', '" + name + "')";
```

Comment:
There is no use case defined for a param named values. req.params.values will always be null.